src/NuGet.Clients/NuGet.PackageManagement.UI/Models/IText.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+
+using System;
+
+namespace NuGet.PackageManagement.UI
+{
+    public interface IText
+    {
+        string Text { get; }
+    }
+
+    internal class LicenseText : IText
```

Comment:
I would suggest to have `LicenseText`, `FreeText` and `WarningText` in their own file for better discovery of the code. I don't have an strong opinion, so if you feel strongly against it disregard this comment.